### ISSUE TITLE
The attack accuracy on Cifra10

### ISSUE THREAD

**Issue by TIEHua** [Harry24k/adversarial-attacks-pytorch#38]
Hi! I trained resnet18 on the Cifra10 and used FGSM to generate adversarial images.
But I found that the accuracy didn't decrease significantly when the eps was set at **(2/255,4/255,8/255)**.
The accuracy on the clean images is 93%. When I add FGSM, the accuary is: **55.8%** for **eps=2/255, 55.3% for eps=4/255, 53.5% for 8/255**.  
I think when the eps changes, the accuracy should drop drastically. This test result makes me feel a little puzzled。

**Comment by Harry24k** [comment]
Hello! When I trained Wide-RestNet-28-10 (WRN28-10) with standard training, it showed the standard accuracy of 96.26%. Against FGSM, it showed the robust accuracy of 64.43%, 55.48%, and 47.35% for eps=2/255, 4/255, 8/255, respectively.

In the case of adversarially trained model, it showed the robust accuracy of 87.07%, 81.01%, and 69.34% for eps=2/255, 4/255, 8/255, respectively.

It can be different when the different training methods or different model structures (e.g., dropout) are used. You can share your training method or the trained model for the further analysis :)

**Comment by TIEHua** [comment]
Thanks for your reply. I used ResNet18 model without dropout. The loss function I used CrossEntropy loss. I trained the model on the Cifra10 and attack the model in inference stage. 
I also found when I decreased the **eps** to a very low value, the accuracy is still **50%-60%**.
The code is showing as below:

  model.eval()
  atk=FGSM(model,eps=0.001)
   for i, (input, target) in enumerate(val_loader):
        #with torch.no_grad():
        input=atk(input,target)
        pdb.set_trace()
        target = target.cuda()
        input = input.cuda()    
        # Compute output
        output= model(input)`

**Comment by Harry24k** [comment]
Does input is scaled to [0, 1]?

**Comment by TIEHua** [comment]
Thanks a lot. I used mean and std to do the normalization. Maybe it's the problem. I will check.

**Comment by TIEHua** [comment]
So I need scale the input to [0,1] and then generate adversarial examples.   And adversarial examples needs to be normalization before inputting the model.

**Comment by Harry24k** [comment]
You can use a normalize layer instead of transforms. Please refer to https://nbviewer.jupyter.org/github/Harry24k/adversarial-attacks-pytorch/blob/master/demos/White%20Box%20Attack%20%28ImageNet%29.ipynb

**Comment by TIEHua** [comment]
Thanks you! I will check my code.
